python/paddle/fluid/io.py

```diff
@@ -1526,8 +1536,76 @@ def load(program, model_path, executor=None):
     assert executor is None or isinstance(executor, Executor)
 
     parameter_file_name = model_path + ".pdparams"
-    assert os.path.exists(parameter_file_name), \
-        "Parameter file [{}] not exits".format(parameter_file_name)
+    #assert os.path.exists(parameter_file_name), \
+    #    "Parameter file [{}] not exits".format(parameter_file_name)
+
+    if not os.path.exists(parameter_file_name):
+        # parameter file not exist, try to load previous save inference
+        # (save_vars, save_params, save_persistables)
+        # get many binary files format
+        _logger.warning(
+            "{} not found, try to load model file save by [ save_params, save_persistables, save_vars ]".
+            format(parameter_file_name))
+        if executor is None:
+            raise ValueError(
+                "When load model file save by [ save_params, save_persistables, save_vars ], executor CAN NOT be None"
```

Comment:
executor is required hen loading model file saved with...

Reply:
done